tailwind.config.js

```diff
@@ -2,7 +2,7 @@ const { url } = require('inspector');
 
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{js,jsx,ts,tsx}"],
+  content: ["./src/**/*.{js,jsx,ts,tsx}", './src/**/*.{html,js}', './node_modules/tw-elements/dist/js/**/*.js'],
```

Comment:
just was wondering what is this for ?

Reply:
this also applies to the tw-elements, but I believe I was looking for an element that would get create the checkbox slider at the bottom, I remember just importing that randomly, turns out adding/removing those imports doesn't make a different though

Reply:
ok turns out I can't just remove them otherwise the vercel breaks, and I need those imports otherwise the slider doesn't render correctly